File: WebAppVen/Scripts/app/model/UrInid.js
Ext.define('WEBAPP.model.UrInid', {
    extend: 'Ext.data.Model',
    fields: [
        { name: 'INID', type: 'string' },
        { name: 'INID_NAME', type: 'string' },
        { name: 'INID_OLD', type: 'string' },
        { name: 'INID_FLAG', type: 'string' },
        { name: 'INID_FLAG_NAME', type: 'string' },
        { name: 'CREATE_TIME', type: 'date' },
        { name: 'CREATE_USER', type: 'string' },
        { name: 'UPDATE_TIME', type: 'date' },
        { name: 'UPDATE_USER', type: 'string' },
        { name: 'UPDATE_IP', type: 'string' }
    ]
});

File: WebApp/Models/UR/UR_INID.cs
using System;
using System.Collections.Generic;

namespace WebApp.Models
{
    public class UR_INID : JCLib.Mvc.BaseModel
    {
        public string INID { get; set; }
        public string INID_O { get; set; }
        public string INID_NAME { get; set; }
        public string INID_OLD { get; set; }
        public string INID_FLAG { get; set; }
        public string INID_FLAG_NAME { get; set; }
        public string GRP_NO { get; set; }
        public DateTime CREATE_TIME { get; set; }
        public string CREATE_USER { get; set; }
        public string CREATE_USER_NAME { get; set; }
        public DateTime? UPDATE_TIME { get; set; }
        public string UPDATE_USER { get; set; }
        public string UPDATE_USER_NAME { get; set; }
        public string UPDATE_IP { get; set; }
    }
}